frontend/src/components/dashboard/DatasetsStats.tsx

```diff
@@ -26,20 +26,24 @@ export default function DatasetsStats() {
     {
       id: 1,
       name: "All",
-      stat: datasets?.length ?? 0,
+      stat: datasets?.results.length ?? 0,
```

Comment:
This will raise an exception if datasets is undefined.

```suggestion
      stat: datasets?.results?.length ?? 0,
```

Reply:
this file is not being used. @demenech